cabal.project

```diff
@@ -174,8 +174,8 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-ledger-specs
-  tag: 22f53f75e80a70243bd0fa41f570a9a2715d76cc
-  --sha256: 033qarxg1cifv1y7km170q08idznhygi3xwi3dral5cchi3gdwgm
+  tag: 7540043e0c565896be4eb8b278587e95eefa8a95
```

Comment:
This pulls in input-output-hk/cardano-ledger-specs#2283, which means we don't have to declare any orphan instances.